### PR TITLE
[zh] Sync 1.20 upgrade note into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync 1.20 upgrade note into Chinese

## Reviewers

@windsonsea 

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  从而很容易意外地将端口上的所有流量发送到错误的位置。
+* 由于目标 DNS 名称被视为不透明，因此我们无法按预期对其应用 Istio 策略。
+  例如，如果外部名称指向另一个集群内服务
+  （例如 `example.default.svc.cluster.local`），则 mTLS 也不会被应用。
+
+对 `ExternalName` 的支持进行了重构，用于解决这些问题。
+`ExternalName` 现在被简单地视为别名。无论我们在哪里匹配 `Host: <concrete service>`，
+我们都会另外匹配 `Host: <external name service>`。
+请注意，`ExternalName` DNS 的主要实现是在 Kubernetes DNS
+实现中的 Istio 外部被处理的，并将保持不变。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:43]:
```suggestion
请注意，`ExternalName` DNS 的主要实现是在 Istio 外部的
Kubernetes DNS 实现中处理的，并将保持不变。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+如果您将 `ExternalName` 与 Istio 一起使用，请注意以下行为变化：
+
+* 不再需要 `ports` 字段，以匹配 Kubernetes 行为。
+  如果设置了该字段，将不会产生任何影响。
+* 在 `ExternalName` 服务上匹配的 `VirtualServices` 一般情况下将不再匹配。
+  相反，应该将匹配重写为被引用的服务。
+* `DestinationRule` 不再适用于 `ExternalName` 服务。
+  相反，创建规则通过 `host` 引用服务。
+
+这些变更在此版本中默认处于关闭状态，但在不久的将来将默认处于打开状态。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:54]:
```suggestion
这些变更在此版本中默认处于关闭状态，但在不久的将来将默认开启。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+需要尽早选择开启，可以设置 `ENABLE_EXTERNAL_NAME_ALIAS=true` 环境变量。
+
+## Envoy 过滤器排序 {#envoy-filter-ordering}
+
+此更改会影响 Envoy 过滤器排序方式的内部实现。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:59]:
```suggestion
此更改影响 Envoy 过滤器排序方式的内部实现。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+这些按照顺序运行的过滤器是为了实现各类功能。
+
+现在，入站、出站和网关代理模式以及 HTTP 和 TCP 协议的顺序是一致的：
+
+* 元数据交换
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:64]:
```suggestion
* Metadata Exchange
```

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+* CUSTOM Authz
+* WASM Authn
+* Authn
+* WASM Authz
+* Authz
+* WASM 统计
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:70]: 这里的几个 filter 我觉得可以不翻译

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+* 统计
+* WASM 未指定
+
+这改变了以下领域：
+
+* 入站 TCP 过滤器现在将元数据交换置于 Authn 之前。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:76]: 同上，Metadata Exchange

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md (hunk 5 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md:70]: 收到

File: content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md (hunk 4 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.20.x/announcing-1.20/upgrade-notes/index.md]: Done

**Review by hanxiaop — APPROVED** [review]
Thanks!

**Review by windsonsea — APPROVED** [review]
Thanks

**Pull request merged**